net/shorewall6/Makefile

```diff
@@ -0,0 +1,114 @@
+#
```

Comment:
I don't think you need two different projects.  See how `isc-dhcp` handles building `dhcpd` and `dhcp6d` from the same `Makefile`.

Reply:
Upstream has split shorewall in seperate source-packages. IPv6 is an different tar.gz then IPv4.
So combining it into one package is not logical.

Reply:
Hello Philip,

Can you review #4486 again?
Thanks

/Willem

Reply:
Different package, separate commit. You get it by now ;-)

Reply:
> Upstream has split shorewall in seperate source-packages. IPv6 is an different tar.gz then IPv4.
> So combining it into one package is not logical.

For what it’s worth, packages like usb-utils is built from 2 tarballs and so are some others.

That said, someone might want IPv4 but not v6 or vice versa.